### PR TITLE
Remove non-blog-related content

### PR DESCRIPTION
**Pull request by dopieralad**
Blog after changes:
image

### CODE REVIEW

**Comment by bgalek** [comment]
GTLM! :D

File: _posts/2019-04-01-from-php-to-java-and-back-again.md [hunk 1]
```diff
@@ -7,7 +7,7 @@ tags: [tech, php, java, monolith, microservices]
 
 > This post was published on April 1st, 2019, and should not be taken too seriously.
 
-Five years ago we started a major technological transformation of the [Allegro](/about-us/) platform.
+Five years ago we started a major technological transformation of the [Allegro](https://allegro.tech/about-us/) platform.
```
Comment (bgalek) [inline _posts/2019-04-01-from-php-to-java-and-back-again.md:10]: @dopieralad what do you think about using value from .config file ("url") or `../about-us/`? :)

**Review by bgalek — APPROVED** [review]

File: _posts/2019-04-01-from-php-to-java-and-back-again.md (hunk 1 above)
Comment (mkosmul) [reply _posts/2019-04-01-from-php-to-java-and-back-again.md]: actually, it should be {{site.url}}{{site.baseurl}}/

File: _posts/2014-12-09-How-to-migrate-to-Java-8.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ using Java 8 will become more of a necessity. It is better to be ahead of the cr
 We also expected gains in productivity from new features such as lambda expressions, so reaping these benefits reasonably early seemed
 like a good idea.
 
-You may wonder why we wanted to spend time on Java 8 with Scala being around. At [Allegro](/about-us/), development teams
+You may wonder why we wanted to spend time on Java 8 with Scala being around. At [Allegro]({{ '/about-us' | prepend: site.url }}), development teams
```
Comment (mkosmul) [inline _posts/2014-12-09-How-to-migrate-to-Java-8.md:21]: better use {{site.baseurl}}{% link /about-us/ %} in places like this

**Review by mkosmul — COMMENTED** [review]
Zmieniłbym sposób generowania linków do /about-us/, jak w komentarzach

File: _posts/2014-12-09-How-to-migrate-to-Java-8.md (hunk 2 above)
Comment (bgalek) [reply _posts/2014-12-09-How-to-migrate-to-Java-8.md:21]: @mkosmul I'll merge this and fix it :)

**Pull request merged**